cinema_service/urls.py

```diff
@@ -6,4 +8,4 @@
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
Ensure that the `cinema.urls` module has a `app_name` defined to support the use of the `namespace` argument in the `include` function.